dispatcher.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env sh
 
 if [ $# -lt 1 ]; then
+    printf "usage: phanes TEMPLATENAME [ ARGS ... ]\n"
```

Comment:
Echo includes a new line by default
```suggestion
    echo "usage: phanes TEMPLATENAME [ ARGS ... ]"
```

Reply:
`echo` is not POSIX portable. See "APPLICATION USAGE": https://pubs.opengroup.org/onlinepubs/9699919799/utilities/echo.html